jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java

```diff
@@ -42,7 +42,7 @@ public class AngularPage extends WebPage {
     public static FormFieldsSection formFieldsSection;
     public static SideNavSection sidenavSection;
     public static ExpansionPanelSection expansionPanelSection;
-    public static SpinnerSection spinnerSection;
+    public static ProgressSpinnerSection spinnerSection;
     public static SortingOverviewSection sortingOverviewSection;
     public static ChipsSection chipsSection;
     public static TableSection tableSection;
```

Comment:
Please, have a look into Katya Sadova PR. Let's choose together versions of naming section vs page. And whether to use NewAngularPage. To avoid conflicts :)

Reply:
I will change it to ProgressSpinnerPage as soon as that PR is merged, so there will be one approach with NewAngularPage.